flight/PiOS/posix/inc/pios_posix.h

```diff
@@ -32,6 +32,12 @@
 
 #include <time.h>
 
+#ifdef PIOS_INCLUDE_SIMSENSORS
```

Comment:
This is why MacOS didn't work--- no hardware drivers were built in, triggering an assert in stabilization from having a a baro when none is expected.